public-api/src/main/java/org/pmiops/workbench/cdr/model/QuestionConcept.java

```diff
@@ -0,0 +1,156 @@
+package org.pmiops.workbench.cdr.model;
+
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Entity
+//TODO need to add a way to dynamically switch between database versions
+//this dynamic connection will eliminate the need for the catalog attribute
+@Table(name = "concept")
+public class QuestionConcept {
+
+    private long conceptId;
+    private String conceptName;
+    private String conceptCode;
+    private String domainId;
+    private long countValue;
+    private float prevalence;
+    private List<AchillesAnalysis> analyses = new ArrayList<>();
+
+    @Id
+    @Column(name = "concept_id")
+    public long getConceptId() {
+        return conceptId;
+    }
+
+    public void setConceptId(long conceptId) {
+        this.conceptId = conceptId;
+    }
+
+    public QuestionConcept conceptId(long conceptId) {
+        this.conceptId = conceptId;
+        return this;
+    }
+
+    @Column(name = "concept_name")
+    public String getConceptName() {
+        return conceptName;
+    }
+
+    public void setConceptName(String conceptName) {
+        this.conceptName = conceptName;
+    }
+
+    public QuestionConcept conceptName(String conceptName) {
+        this.conceptName = conceptName;
+        return this;
+    }
+
+    @Column(name = "concept_code")
+    public String getConceptCode() {
+        return conceptCode;
+    }
+
+    public void setConceptCode(String conceptCode) {
+        this.conceptCode = conceptCode;
+    }
+
+    public QuestionConcept conceptCode(String conceptCode) {
+        this.conceptCode = conceptCode;
+        return this;
+    }
+
+    @Column(name = "domain_id")
+    public String getDomainId() {
+        return domainId;
+    }
+
+    public void setDomainId(String domainId) {
+        this.domainId = domainId;
+    }
+
+    public QuestionConcept domainId(String domainId) {
+        this.domainId = domainId;
+        return this;
+    }
+
+    @Column(name= "count_value")
+    public long getCountValue() {
+        return countValue;
+    }
+
+    public void setCountValue(long count) {
+        this.countValue = count;
+    }
+
+    public QuestionConcept count(long count) {
+        this.countValue = count;
+        return this;
+    }
+
+    @Column(name = "prevalence")
+    public float getPrevalence() {
+        return prevalence;
+    }
+
+    public void setPrevalence(float prevalence) {
+        this.prevalence = prevalence;
+    }
+
+    public QuestionConcept prevalence(float prevalence) {
+        this.prevalence = prevalence;
+        return this;
+    }
+
+    /*@OneToMany(fetch = FetchType.EAGER, orphanRemoval = true, cascade = CascadeType.ALL)
```

Comment:
Should we have commented out code in a PR?

Reply:
Not that code :) Thanks . I'm not against it on principle necessarily but I just missed that on clean up.